request.go

```diff
@@ -18,6 +18,10 @@ type Request struct {
 	Dst string
 	Pwd string
 
+	// Forced getter detected in Src during Getter detection phase
+	// This is currently for internal use only
+	Forced string
+
```

Comment:
Ah, I wish things around that `Forced` field where a little less funky. 

Can you please document a little more things around this field ? I keep forgetting why we need that field 😓 .